package.json

```diff
@@ -34,5 +34,8 @@
     "path": "^0.12.7",
     "config-js": "^1.1.9",
     "express": "^4.13.4"
+  },
+  "dependencies": {
+    "promise": "^7.1.1"
```

Comment:
This dependency doesn't seem to be used in the PR. I don't see anywhere it's required.